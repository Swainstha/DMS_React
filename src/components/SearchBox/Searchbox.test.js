import testSnapFunction from '../../util/testSnapFunction';
import SearchBox from './index'
import React from 'react';
const dummy_data= {
 styleName:"style1", 
 placeholder:"search", 
 onChange:jest.fn(), 
 value:"value"
}

testSnapFunction("<SearchBox />", "Snapshot test for SearchBox ",<SearchBox {...dummy_data}><div>dsjfM</div> </SearchBox> )