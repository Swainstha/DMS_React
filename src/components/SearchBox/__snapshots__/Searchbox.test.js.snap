// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> Snapshot test for SearchBox  1`] = `ShallowWrapper {}`;
